mbuild/tests/test_lammpsdata.py

```diff
@@ -380,9 +381,9 @@ def test_lj_box(self, ethane):
                     # assert np.isclose(line, 0.204)
                     assert np.isclose(float(line), 2.04)
                     line = float(fi.readline().split()[1])
-                    assert np.isclose(line, 0.2268)
+                    assert np.isclose(line, 2.268)
                     line = float(fi.readline().split()[1])
-                    assert np.isclose(line, 0.1898857)
+                    assert np.isclose(line, 1.898857)
```

Comment:
I am not exactly sure if I am correct, but the asserted box lengths in the test seem off here: The first box length is 2.04 which seems ok, but the other two seem 10x smaller than they should be. Under LJ unit, the length unit is the largest sigma of all the atoms (which is opls_135: carbon in this case), and it doesn't make sense to have a box length that is 1/5 of the diameter of carbon. Might originates the (unneeded in `lj` `unit_style`) conversion from A to nm again.